pkg/registry/apiserverinternal/storageversion/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package storageversion provides model implementation of storage-version api
```

Comment:
What does "model implementation" mean?

Reply:
it's copied from [flowcontrol](https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/flowcontrol/flowschema/doc.go) godoc. I updated the comment following the pattern in core API godoc